src/RackspaceAdapter.php

```diff
@@ -162,7 +162,23 @@ public function deleteDir($dirname)
      */
     public function createDir($dirname, Config $config)
     {
-        return ['path' => $dirname];
+        $headers = [];
+
+        if ($config->has('headers')) {
+            $headers = $config->get('headers');
+        }
+
+        if (!key_exists('Contents', $headers)) {
+            $headers['Contents'] = 'None';
```

Comment:
Alternatively, shouldn't it just have no content at all? Or is this in the docs as well?